package.json

```diff
@@ -27,14 +27,14 @@
   "types": "./index.d.ts",
   "dependencies": {},
   "peerDependencies": {
-    "@atomist/automation-client": "*",
-    "@atomist/sdm": "*",
-    "@atomist/sdm-core": "*"
+    "@atomist/automation-client": ">=1.7.0",
```

Comment:
I thought we agreed that we go with caret ranges for peer dependencies too now? Do I recall that wrong?

Reply:
No, you are right. I'll fix that.